Directory.Packages.props

```diff
@@ -16,6 +16,7 @@
     <PackageVersion Include="MediatR" Version="11.1.0" />
     <PackageVersion Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
     <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.21.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
```

Comment:
Do we still need this reference? Or does it contain `ITelemetryClient` and other necessary abstractions?